src/pages/getting-started/app-guides/integrations/index.mdx

```diff
@@ -9,7 +9,7 @@ desc: "Streamline your workflow by integrating with supported development servic
 
 # Introduction
 
-This document is a reference of integrations available in the <Placeholder id="DEVELOPER_CONSOLE_URL" />. Connect with different development tools and streamline your workflow by creating repositories directly from the Console and integrating with supported deployment services (coming soon!).
```

Comment:
@litwinow please do not pay attention to this one, I just needed to remove it so I took a chance I have this PR😜